main/py3-meld3/APKBUILD

```diff
@@ -1,41 +1,33 @@
 # Contributor: Peter Bui <pnutzh4x0r@gmail.com>
 # Maintainer:
-pkgname=py-meld3
+pkgname=py3-meld3
 _pkgname=meld3
 pkgver=1.0.2
-pkgrel=1
+pkgrel=2
 pkgdesc="An HTML/XML templating engine"
 url="https://github.com/supervisor/meld3"
 arch="noarch"
 license="ZPL"
```

Comment:
License is incorrect https://github.com/Supervisor/meld3/blob/master/LICENSE.txt

Reply:
Udated to "Repose" cos bsd-4 is not the same